From: ayaka <ayaka@soulik.info>
Date: Tue, 8 Jul 2014 02:52:11 +0800
Subject: [PATCH] v4l2: modify the max encoded size

For the encoder of s5p-mfc the size is 2MB, but it is correct
for decoder which is 1MB.
---
 sys/v4l2/gstv4l2object.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/v4l2/gstv4l2object.c b/sys/v4l2/gstv4l2object.c
index 37fbc746d977..35eda7230af6 100644
--- a/sys/v4l2/gstv4l2object.c
+++ b/sys/v4l2/gstv4l2object.c
@@ -56,7 +56,7 @@ GST_DEBUG_CATEGORY_EXTERN (v4l2_debug);
 #define DEFAULT_PROP_FREQUENCY          0
 #define DEFAULT_PROP_IO_MODE            GST_V4L2_IO_AUTO
 
-#define ENCODED_BUFFER_SIZE             (1 * 1024 * 1024)
+#define ENCODED_BUFFER_SIZE             (2 * 1024 * 1024)
 
 enum
 {
